modules/fscloud/README.md

```diff
@@ -0,0 +1,67 @@
+# Profile for IBM Cloud Framework for Financial Services
+
+This code is a version of the [parent root module](../../) that includes a default configuration that complies with the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about). See the [Example for IBM Cloud Framework for Financial Services](/examples/fscloud/) for logic that uses this module.
+
+:exclamation: **Exception:** The Databases for RabbitMQ service is not yet Financial services validated. Therefore, the infrastructure that is deployed by this module is also not validated with the Framework for Financial Services. For more information, see the list of [Financial Services Validated services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about#financial-services-validated-services).
+
+The default values in this profile were scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) for compliance with the IBM Cloud Framework for Financial Services profile that is specified by the IBM Security and Compliance Center. The scan passed for all applicable goals with one exception:
+
+> rule-beb7b289-706b-4dc0-b01d-b1d15d4331e3: Check whether RabbitMQ network access is restricted to a specific IP range.
```

Comment:
Is this correct? I don't see this rule being ignored in `cra-tf-validate-ignore-rules.json` and I don't see it failing in the pipeline either?

Reply:
@Aayush-Abhyarthi I don't see this rule failing - can you confirm?

Reply:
@Aayush-Abhyarthi There is no failing rule in the CRA scan - why is still documented as a failing rule?